lib/dbi/ogs-dbi.h

```diff
@@ -25,7 +25,7 @@
 #define OGS_DBI_INSIDE
 
 /* TODO: remove ogs-mongoc.h as soon the test cases don't depend on it */
-#include "dbi/ogs-mongoc.h"
+#include "dbi/mongo/ogs-mongoc.h"
```

Comment:
the meson file seems to indicate you make the mongo files optional, but you are including it here non-conditionally?